weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/traceServerClientTypes.ts

```diff
@@ -124,6 +124,7 @@ export type TraceCallsQueryReq = {
   include_costs?: boolean;
   include_feedback?: boolean;
   include_total_storage_size?: boolean;
+  include_storage_size?: boolean;
```

Comment:
Doesn't really matter but maybe move this above include_total_storage_size for consistent ordering between files.